task14/task14.js

```diff
@@ -0,0 +1,32 @@
+function TextCell(text) {
+    this.text = text.split("\n");
+}
+
+TextCell.prototype.minWidth = function () {
+    return this.text.reduce(function (width, line) {
+        return Math.max(width, line.length);
+    }, 0);
+};
+
+TextCell.prototype.minHeight = function () {
+    return this.text.length;
+};
+
```

Comment:
Add both _TextCell.prototype.draw_ and _StretchCell.prototype.draw_ functionality from the topic
With console log check for it

Reply:
:_(